docker/entrypoint/docker-entrypoint.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+if [ ! -e node_modules ]; then
+#Running the installer, hold on to your butts.
+echo "---"
+echo "---"
+echo "#Running the installer, hold on to your butts."
+echo "---"
+echo "---"
+npm install
```

Comment:
can we use `yarn install` here instead? generally it's not a good idea to mix npm and yarn. I think alpine-node should have yarn pre-installed as well.